docs/stdlib/datetime.rst

```diff
@@ -827,54 +827,52 @@ EdgeDB stores and outputs timezone-aware values in UTC.
                   std::duration_get(dt: cal::date_duration, \
                                     el: str) -> float64
 
-    Extract a specific element of input duration by name.
+    Returns the element of a duration given a unit name.
 
     .. note::
 
       Only available in EdgeDB 2.0 or later.
 
-    There units avaialble for extraction are grouped into 3 categories. The
-    largest units category of months or greater:
+    You may pass any of these unit names as ``el``:
 
     - ``'millennium'`` - number of 1000-year chunks rounded down
-    - ``'century'`` - the number of centiries rounded down
-    - ``'decade'`` - the number of decades rounded down
-    - ``'year'`` - the numbder of years rounded down
-    - ``'quarter'``- the quarter left over after whole years are
+    - ``'century'`` - number of centuries rounded down
+    - ``'decade'`` - number of decades rounded down
+    - ``'year'`` - number of years rounded down
+    - ``'quarter'``- remaining quarters after whole years are accounted for
+    - ``'month'`` - number of months left over after whole years are
       accounted for
-    - ``'month'`` - the number of months left over after whole years are
-      accounted for
-
-    The middle size units of days:
-
-    - ``'day'`` - specifically the number of days recorded in the duration
-
-    The smallest unit category of hours, minutes, seconds, etc.:
-
-    - ``'hour'`` - duration hours
-    - ``'minutes'`` - duration minutes (0-59)
-    - ``'seconds'`` - duration seconds, including fractional value from 0 up
-      to and not including 60
-    - ``'milliseconds'`` - the seconds including fractional value expressed
-      as milliseconds
-    - ``'microseconds'`` - the seconds including fractional value expressed
-      as microseconds
+    - ``'day'`` - number of days recorded in the duration
+    - ``'hour'`` - number of hours
+    - ``'minutes'`` - remaining minutes after whole hours are accounted for
+    - ``'seconds'`` - remaining seconds, including fractional value after whole
+      minutes are accounted for
+    - ``'milliseconds'`` - remaining seconds including fractional value
+      expressed as milliseconds
+    - ``'microseconds'`` - remaining seconds including fractional value
+      expressed as microseconds
+
+    .. note ::
+
+      Only for units ``'month'`` or larger or for units ``'hour'`` or smaller
+      will you receive a total across multiple units expressed in the original
+      duration. See *Gotchas* below for details.
 
     Additionally, it's possible to convert a given duration into seconds:
 
     - ``'totalseconds'`` - the number of seconds represented by the
       duration. It will be approximate for :eql:type:`cal::relative_duration`
-      and :eql:type:`cal::date_duration` with usints larger than a day,
+      and :eql:type:`cal::date_duration` for durations longer than a day,
```

Comment:
It's approximate if the duration included units of months or larger, not just "longer than a day". As long as the units are just days, the `totalseconds` conversion will simply assume 24hr days and do an exact conversion. Approximations happen for larger units: months+.